tests/improver-nowcast-extrapolate/02-basic.bats

```diff
@@ -33,38 +33,25 @@
 
 @test "extrapolate basic" {
   improver_check_skip_acceptance
-  KGO0="nowcast-extrapolate/extrapolate/kgo0.nc"
-  KGO1="nowcast-extrapolate/extrapolate/kgo1.nc"
-  KGO2="nowcast-extrapolate/extrapolate/kgo2.nc"
+  KGO="nowcast-extrapolate/extrapolate/kgo.nc"
 
-  UCOMP="$IMPROVER_ACC_TEST_DIR/nowcast-optical-flow/basic/ucomp_kgo.nc"
-  VCOMP="$IMPROVER_ACC_TEST_DIR/nowcast-optical-flow/basic/vcomp_kgo.nc"
+  UVCOMP="$IMPROVER_ACC_TEST_DIR/nowcast-optical-flow/basic/kgo.nc"
```

Comment:
This file has been recreated in your new test data but matches the data already in the archive.

Reply:
fixed.